build/Targets/Imports.targets

```diff
@@ -453,6 +453,13 @@
                                TargetMonikers=".NETFramework,Version=v4.6">
       <Output TaskParameter="ResolvedCopyLocalItems" ItemName="ReferenceCopyLocalPaths" />
     </ResolveNuGetPackageAssets>
+
+    <MSBuild Projects="$(MSBuildThisFileDirectory)..\..\src\Test\Utilities\Desktop\TestUtilities.Desktop.csproj"
+             Targets="GetTargetPath"
+             BuildInParallel="true">
+
+        <Output TaskParameter="TargetOutputs" ItemName="ReferenceCopyLocalPaths" />
+    </MSBuild>
```

Comment:
I was wondering how you would work around this problem.  Can you add a comment documenting what this accomplishes?